docs/Concepts/Pruning.md

```diff
@@ -12,8 +12,8 @@ latest state for the blockchain (and some older states, but not all). You can ch
 balances, sign and send transactions, and look at current Dapp data.
 
 Archive nodes have all of this and they also store the intermediary state of every account and
-contract for every block since the genesis block. With an archive node you can do everything you
-do with a full node, as well as access historical state data.
+contract for every block since the genesis block.
+With an archive node you can do everything you do with a full node and also access historical state data.
```

Comment:
Suggestion, "An archive can do everything a full node does, and it can access historical state data."